packages/@aws-cdk/aws-ec2/lib/vpc-peering-connection.ts

```diff
@@ -0,0 +1,132 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import { CfnVPCPeeringConnection } from './ec2.generated';
+import { IVpc } from './vpc';
+
+/**
+ * VPC with which you are creating the VPC peering connection.
+ */
+export class PeerVpc {
```

Comment:
I think that if you wait for https://github.com/aws/aws-cdk/pull/8280 to be merged, this can be replaced with an `IVpc` instead which seems nicer.

Reply:
Well, the peer vpc is different from a regular vpc.

- You cannot get its details. only `account`, `region`, `vpc-id`, and `cidr` are provided.
- You have no control over it. You cannot do anything with it. So `IVpc` might be false implication.